src/_includes/layouts/base/index.njk

```diff
@@ -18,6 +18,9 @@
     {% include "shared/svgs.njk" %}
 
     <main class="main{% if templateClass %} {{ templateClass }}{% endif %}">
+      {% if templateClass === "entry" %}
```

Comment:
All ears if there's a better way/place to put this!

Reply:
makes sense to me, and is clear to read, so I'm good with it! cc @bchiang7